linode/manager
# M3-5499: Cleanup TextField story

## Description

- Remove props that are no longer in use or have no effect (ie. `affirmative`, `small`, and `tiny`).
- Convert to a function component (required in order to import the props to the `ArgsTable` in the docs)
- Synchronize the naming convention for the `Button` docs and add default values to the `ArgsTable`

## How to test

- Check the text fields throughout the app
- Run `yarn storybook` to see the documentation for TextFields (it's nested inside the `Components`/`Forms` folder)
